common/membership/ringpop_config.go

```diff
@@ -287,11 +242,10 @@ func (provider *dnsSRVProvider) Hosts() ([]string, error) {
 	var results []string
 	resolvedHosts := map[string][]string{}
 
-	for _, service := range provider.UnresolvedHosts {
-		serviceParts := strings.Split(service, ".")
+	for _, host := range provider.UnresolvedHosts {
+		serviceParts := strings.Split(host, ".")
```

Comment:
nit: maybe this should be `hostParts` as well. I find whole section quite confusing.

Reply:
thanks, will update